pkg/operation/cloudbotanist/openstackbotanist/controlplane.go

```diff
@@ -61,7 +61,74 @@ func (b *OpenStackBotanist) GenerateKubeSchedulerConfig() (map[string]interface{
 }
 
 // GenerateEtcdBackupConfig returns the etcd backup configuration for the etcd Helm chart.
-// TODO: implement backup functionality for OpenStack
 func (b *OpenStackBotanist) GenerateEtcdBackupConfig() (map[string][]byte, map[string]interface{}, error) {
-	return nil, nil, nil
+	containerName := "containerName"
+	authURL := "authURL"
```

Comment:
For the sake of completeness, can you define that as a constant here: https://github.com/gardener/gardener/blob/master/pkg/operation/cloudbotanist/openstackbotanist/types.go#L25 ?